src/microbe_stage/compound_absorber_system.cpp

```diff
@@ -150,7 +156,8 @@ void
         const auto grabRadius = membrane.calculateEncompassingCircleRadius();
 
         // This version is used when working with cloud local coordinates
-        const auto localGrabRadius = grabRadius / CLOUD_RESOLUTION;
+        const auto localGrabRadius =
+            grabRadius / CLOUD_RESOLUTION / absorber.scale;
```

Comment:
I think having `* absorber.scale` here would be more intuitive, at least for me.

Reply:
this is also now done